packages/react/src/components/AnimatedSprite.js

```diff
@@ -28,6 +28,17 @@ const AnimatedSprite = (root, props) =>
 
         let changed = applyDefaultProps(instance, oldProps, props);
 
+        if (images && oldProps.images !== images)
```

Comment:
We need to add `images` to line 27 so that we're grabbing the new images prop out of `props`, and not sending the `images` prop to `applyDefaultProps()`.